Automattic/jetpack
# Gutenpack: Add support for feature limited blocks

This adds support for adding features to blocks which are limited based on (for example) the plan that the site is on.




#### Changes proposed in this Pull Request:

* Add a `limited` attribute to the block when it's registered
* The `limited` attribute is added to the block availability data
* This can then be used (as in the example in this PR) to display some features dependent on the plan of the site (or some other factor).

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of pb5gDS-62-p2

#### Testing instructions:






* For a site on a free plan...
* Add a contact form
* Check that there is no "Google Sheets Integration" option in the block sidebar
* Upgrade the site to any other Jetpack plan
* Check that the  "Google Sheets Integration" option in the block sidebar appears

#### Proposed changelog entry for your changes:

* Add support for feature limited blocks
